.qlty/configs/.shellcheckrc

```diff
@@ -0,0 +1 @@
+source-path=SCRIPTDIR
```

Comment:
Please add empty line 😱😱

Reply:
To be precise, need to add a `\n` so that there is one canonical line in the file. To add an empty line would mean to add another `\n`. This is how I understood text file spec awhile ago. Who would have thought free text files also have formatting rules..

But I don't care whether we are standards compliant with this.